README.md

```diff
@@ -52,3 +98,7 @@ It's pretty strait forward.
 
 If you want, you can add a pre-build script to your project to get the manifest's "android:versionName" and set it right into the String.xml file, but that's up to you.
 
+## Known Issues
+The targeted Xamarin.Android project platform needs to be of Android 8.0 otherwise the application is not able to initialize with firebase.
```

Comment:
needs to target Android 8.0 (API Level 26). 

I think the word target makes it clearer that we're talking about compilation and not the device.

Reply:
I agree, i added a better comment